vector/v.vect.stats/testsuite/test_vect_stats.py

```diff
@@ -399,6 +476,535 @@ def test_mode(self):
         self.assertModule(v_vect_stats)
         self.assertLooksLike(reference=output6, actual=v_vect_stats.outputs.stdout)
 
+        # Repeat with explicit plain format
+        v_vect_stats = SimpleModule(
+            "v.vect.stats",
+            points=self.input,
+            areas=self.areas,
+            method="mode",
+            count_column="num_points",
+            stats_column="avg_elev",
+            points_column="cat",
+            flags="p",
+            format="plain",
+        )
+        self.assertModule(v_vect_stats)
+        self.assertLooksLike(reference=output6, actual=v_vect_stats.outputs.stdout)
+
+    def test_sum_csv(self):
+        """Testing method sum with CSV output format"""
+        v_vect_stats = SimpleModule(
+            "v.vect.stats",
+            points=self.input,
+            areas=self.areas,
+            method="sum",
+            count_column="num_points",
+            stats_column="avg_elev",
+            points_column="cat",
+            flags="p",
+            format="csv",
+        )
+        self.assertModule(v_vect_stats)
+        self.assertLooksLike(
+            reference=output1.replace("|", ","), actual=v_vect_stats.outputs.stdout
+        )
+
+    def test_average_csv(self):
+        """Testing method average with CSV output format"""
+        v_vect_stats = SimpleModule(
+            "v.vect.stats",
+            points=self.input,
+            areas=self.areas,
+            method="average",
+            count_column="num_points",
+            stats_column="avg_elev",
+            points_column="cat",
+            flags="p",
+            format="csv",
+        )
+        self.assertModule(v_vect_stats)
+        self.assertLooksLike(
+            reference=output2.replace("|", ","), actual=v_vect_stats.outputs.stdout
+        )
+
+    def test_median_csv(self):
+        """Testing method variance with CSV output format"""
+        v_vect_stats = SimpleModule(
+            "v.vect.stats",
+            points=self.input,
+            areas=self.areas,
+            method="variance",
```

Comment:
Here and also below there is some confusion with the method, function name and column.